lab/goals.md

```diff
@@ -0,0 +1,26 @@
+# OPI Lab Goals
+
+The OPI members may contribute some hardware
```

Comment:
I believe the OPI project can also just purchase cards themselves directly, as nothing prevents that other than card availability.

Reply:
Agree. While vendors are free to contribute, and may be able to contribute cards that are yet to be released, beta, etc. the budget for the lab as originally conceived was large enough for OPI to purchase servers and DPU/IPUs, and to have a lab managed such that those servers and cards could be mixed and matched by smart hands as needed. The original lab discussions were centered around allowing developers to check out a system and card combo for exploratory dev purposes.  I see above a lot of discussion about vendors not wanting others to access their setups for competitive bakeoffs. I totally understand that. Vendors can put those restrictions on any equipment they donate to the project, but to restrict the overall use of a lab, and to restrict the goals of the lab is going too far in my opinion.